### ISSUE TITLE
Sketch: Text nodes disappear temporarily on move

### ISSUE THREAD

**Issue by jacobwod** [hajkmap/Hajk#1225]
**Describe the bug**
The move interaction for text nodes is not super-intuitive. There is probably a good reason why it works as it does, but can we try to do something about it?

https://user-images.githubusercontent.com/110222/197767421-4e6bfba0-36e3-410e-860e-95d3cb6315bc.mov

**Comment by Hallbergs** [comment]
It is OpenLayers standard style used within the move interaction. Should be easy to override!

**Comment by jacobwod** [comment]
I've had a look and the main issue is that it's hard to extend an existing OL style. We either use the default edit style (current situation) or we provide a new one. I can't find the actual default edit style though - OL docs [state here](https://openlayers.org/en/latest/apidoc/module-ol_style_Style-Style.html) a default edit style, but it is not true (e.g. polygons in the docs lack a stroke, but they do have a blue stroke in the real default style as we can see in Hajk). 

I'll see if I can dig into this a bit more, but no quick solution as it looks now.

**Comment by jacobwod** [comment]
This isn't easy, but I have a quick fix which may (or may not) be better. Please review these two options:

### Current situation
Features get another style during move interaction. This is mostly nice, except for text nodes, which just disappear. 

https://user-images.githubusercontent.com/110222/199742586-b8ed233a-f043-4e9b-aa55-5c3959bf0c09.mov

### Alternative solution
Features' style remains unchanged during move interaction. This is not as nice for the polygon, but we still have a small indication about this interaction: mouse cursor is changed to `hand`. The big win here is the edit interaction and what happens with the text node: we can edit and review the changes while we type!

https://user-images.githubusercontent.com/110222/199743515-2f277bfd-ed03-4e2e-b19d-61147c44be32.mov

If everyone's fine with this, I prefer the alternative solution. Please let me know.

**Comment by Hallbergs** [comment]
I prefer the alternative with no "move-styling" as well! As you said, the pointer cursor should be enough.
